eessi_bot_job_manager.py

```diff
@@ -250,34 +252,23 @@ def process_new_job(self, new_job):
             # (b) find & get comment for this job
             # only get comment if we don't know its id yet
             if "comment_id" not in new_job:
-                comments = pr.get_issue_comments()
-                for comment in comments:
-                    # NOTE adjust search string if format changed by event
-                    #        handler (separate process running
-                    #        eessi_bot_event_handler.py)
-                    cms = f".*submitted.*job id `{new_job['jobid']}`.*"
-
-                    comment_match = re.search(cms, comment.body)
-
-                    if comment_match:
-                        log(
-                            "process_new_job(): found comment with id %s"
-                            % comment.id,
-                            self.logfile,
-                        )
-                        new_job["comment_id"] = comment.id
-                        break
+                # search_pattern = f"submitted.*job id `{new_job['jobid']}`"
+                new_job_cmnt = get_comment(pr, new_job['jobid'])
```

Comment:
If parameters to function `get_comment` are changed to `get_comment(pr, search_pattern)` the call here needs to be changed to.